src-java/northbound-service/northbound/src/main/java/org/openkilda/northbound/service/impl/SwitchServiceImpl.java

```diff
@@ -274,7 +272,10 @@ public CompletableFuture<SwitchValidationResult> validateSwitch(SwitchId switchI
         logger.info("Validate request for switch {}", switchId);
 
         return performValidateV2(
-                SwitchValidateRequest.builder().switchId(switchId).processMeters(true).build())
+                SwitchValidateRequest.builder()
+                        .switchId(switchId)
+                        .validationFilters(Arrays.stream(ValidationFilter.values()).collect(Collectors.toSet()))
```

Comment:
Do v1 api need for example "flow_info" filter entry?

And what about `validateRules` method above? It produce same request, but do not raise `processMeters`... In current setup it looks broken, because it must include at least "ValidationFilter.RULES" into filter, as far as I understand.